ui/app/build-configs/detail/revisions-tab/pnc-revisions-details.html

```diff
@@ -23,5 +23,5 @@
       <button type="button" class="btn btn-default" uib-tooltip="Restore current revision" tooltip-placement="top" tooltip-append-to-body="true" tooltip-popup-delay="2500" ng-click="$ctrl.restore()" pnc-requires-auth><i class="pficon pficon-restart"></i> Restore</button>
       </div>
     </div>
-  <pnc-build-config-properties build-config="$ctrl.revision"></pnc-build-config-properties>
+  <pnc-build-config-properties hideFields="description" build-config="$ctrl.revision"></pnc-build-config-properties>
```

Comment:
This should be "hide-fields=" ... I don't think it will work in camel case?

Also does passing the string "description" work? As hideFields is a one way bound field I think it is searching the scope for an object reference called description?

The easiest fix if it doesn't work is to have:
hide-fields="$ctrl.hideFields" here and then in the reveisions details component controllers add: $ctrl.hideFields = ['description'];